engine/src/state_chain_observer/client/subxt_state_chain_config.rs

```diff
@@ -0,0 +1,24 @@
+use subxt::{config::signed_extensions, Config};
+
+pub enum StateChainConfig {}
+
+impl Config for StateChainConfig {
+	type Hash = <state_chain_runtime::Runtime as frame_system::Config>::Hash;
+	type AccountId = subxt::utils::AccountId32;
+	type Address = subxt::utils::MultiAddress<Self::AccountId, ()>;
+	type Signature = subxt::utils::MultiSignature;
+	type Hasher = subxt::ext::sp_runtime::traits::BlakeTwo256;
+	type Header = subxt::ext::sp_runtime::generic::Header<u32, Self::Hasher>;
```

Comment:
If we're creating our own StateChainConfig we might as well explicitly use the values from our Runtime? Ie. `type AccountId = <state_chain_runtime::Runtime as frame_system::Config>::AccountId;` etc. Does this not work?

Reply:
Unfortunately not